AgileCoffeeShop/Coordinate.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace AgileCoffeeShop
+{
+    public class Coordinate
+    {
+        private double _xCoordinate;
+        private double _yCoordinate; 
+        public double XCoordinate
+        {
+            get
+            {
+                return _xCoordinate;
+            } 
+            set
+            {
+                if(value >=-180 && value <= 180)
+                {
+                    _xCoordinate = value;
+                }
+                else
```

Comment:
I do this validation because I researched the coordinate system and there is an interval required.  Maybe I was a little bit wrong but I will do the entire validation for coordinates in the same place.

Reply:
I used a min and a max value because I have done a bit of research about the coordinates system and I have seen that there are some limitations. Also, I've done this validation here just to exemplify that get and set can be extended. I was a little bitt wrong, but I understand that all the validation must be done in one place and I will do the changes.